### ISSUE TITLE
Websites are permanently blocked even after I uninstall SelfControl

### ISSUE THREAD

**Issue by natcatk** [SelfControlApp/selfcontrol#647]
Hi I am a current college student and I downloaded SelfControl in order to block a few website while I am using my laptop to study. I noticed this morning that even though SelfControl was not running, it has continued to block the websites I listed out. I deleted the app from my laptop but these websites continue to be blocked. Does anyone know how to fix this?? I am pretty sure it is the fault of SelfControl because every other website I try to go to works fine, it's only the website I have previously blocked that remain unable to be reached. If you have any advice please help me because I need to get onto one of these websites to do an assignment for school!!

**Comment by cstigler** [comment]
Hi @natcatk, that sounds frustrating! Can you describe exactly what's happening on your computer, in particular what you see when you open up SelfControl? (please re-download it if you've deleted it)

If, when you open up SelfControl, you see a "Finishing" or "00:00" timer window, wait 10 seconds and you'll see a "Stuck? Stop block manually" button appear. You can click that and use it to stop your block. Restart your computer for good measure, and test your access.

If you just see the "Start" window instead, try running another block for 15 minutes and waiting for it to finish. Then restart your computer and test.

If these tips aren't working, feel free to email me at charlie+sc647@charliestigler.com and I can help get you sorted. Please include:

- SelfControl app version and macOS version
- What you're seeing when you open up SelfControl
- What you've already tried (e.g. the suggestions above)
- Whether you're using a VPN/proxy, or if this is a school/corporate laptop
- Any other special circumstances I should know about

**Comment by natcatk** [comment]
Hello!!

I am so sorry I didn't respond to this email sooner, I completely missed
it. I actually was able to figure out the issue I was having. Although the
timer was technically out and I uninstalled SelfControl, all the websites I
listed were still blocked. After I reinstalled SelfControl and removed all
these websites from my list, they stopped being blocked! So I'm sorry for
wasting your time especially when the fix turned out to be very simple. I
also wanted to say that I really appreciate that you emailed me to help me
with this! I was certainly freaking out in the moment before I figured out
what was going on.

Thank you,
Natalie Kim
